.gitignore

```diff
@@ -102,3 +102,5 @@ dist
 
 # TernJS port file
 .tern-port
+
+.history
```

Comment:
What is this .history? Why do we ignore it?

Reply:
what is TernJS?

Reply:
ok

Reply:
because of . it is not saved

Reply:
it is hidden file

Reply:
resolved conversation